classes/File.js

```diff
@@ -104,7 +112,7 @@ class File {
       let params = {
         "message": `Update file: ${fileName}`,
         "content": content,
-        "branch": "staging",
+        "branch": `${this.branchRef}`,
```

Comment:
Remove unnecessary `${}`.

Reply:
Fixed in [fc2f3](https://github.com/isomerpages/isomercms-backend/commit/fc2f3cb1fc67fd143ef4ae2f0feffd6ff5953472)